docker/lb/docker-compose-listeners.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  avalon-listener-1:
+    container_name: avalon-listener-1
+    image: avalon-listener-dev
+    build:
```

Comment:
we could add dependency of  "avalon-listener" in "avalon-listener-1,2,3". This way we can ensure "avalon-listener-dev" image will be available before starting containers for avalon-listener-1,2,3. We can also remove the entire build section in docker-compose-listener file.

Reply:
Done